app/controllers/MovementsController.scala

```diff
@@ -136,8 +136,8 @@ class MovementsController @Inject()(
               case submissionFailureRejected: SubmissionFailureRejected =>
                 BadRequest(submissionFailureRejected.responseBody)
               case SubmissionSuccess =>
-                auditService.auditEvent(AuditType.ArrivalNotificationReSubmitted, request.body)
-                auditService.auditEvent(AuditType.MesSenMES3Added, message.message)
+                auditService.auditEvent(AuditType.ArrivalNotificationReSubmitted, request.body, request.getChannel)
+                auditService.auditEvent(AuditType.MesSenMES3Added, message.message, request.getChannel)
                 Accepted("Message accepted")
                   .withHeaders("Location" -> routes.MovementsController.getArrival(request.arrival.arrivalId).url)
             }
```

Comment:
Possibly not for this PR, but as far as I can tell this map is repeated multiple times and is the same code block. could this functionality be moved to a single function?

Reply:
It took me a second to see what you meant here because of the line selection, but that's a fair point for a refactor. @GeorgeColdham would you write it up as a low-priority tech-debt ticket? Or potentially a low-priority spike?

Reply:
https://jira.tools.tax.service.gov.uk/browse/CTDA-961